ui/src/containers/SolutionList.js

```diff
@@ -135,56 +165,68 @@ const SolutionsList = props => {
     {
       label: intl.translate('description'),
       dataKey: 'description',
-      flexGrow: 1,
     },
     {
       label: intl.translate('solutions'),
       dataKey: 'solutions',
-      renderer: (solutions, row) => {
-        const solutionsLinks = solutions
-          .map((solution, idx) => {
-            const solutionRow = sortedSolutions.find(
-              s => s.name === solution.name,
+      renderer: (solutions, environment) => {
+        const isEnvironmentPreparing = environment.isPreparing;
+        const deployedSolutions = environment.solutions;
+        const solutionsList =
+          deployedSolutions &&
+          deployedSolutions.map((deployedSolution, idx) => {
+            return (
+              <span
+                key={idx}
+              >{`${deployedSolution.name}(v.${deployedSolution.version})`}</span>
```

Comment:
minor: let's show a space after the solution name
```suggestion
              >{`${deployedSolution.name} (v.${deployedSolution.version})`}</span>
```